kotlin-result-coroutines/src/jsTest/kotlin/com.github.michaelbull.result.coroutines/binding/AsyncSuspendableBindingTests.kt

```diff
@@ -0,0 +1,45 @@
+package com.github.michaelbull.result.coroutines.binding
+
+import com.github.michaelbull.result.Err
+import com.github.michaelbull.result.Ok
+import com.github.michaelbull.result.Result
+import com.github.michaelbull.result.coroutines.runBlockingTest
+import kotlinx.coroutines.async
+import kotlinx.coroutines.delay
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class AsyncSuspendableBindingTest {
```

Comment:
I think even if ordering doesn't matter due to JS, it would still be good to add the test here for a failing bind to document the actual behaviour.

Reply:
Absolutely. I added a test for a failing async bind.